README.md

```diff
@@ -1 +1,161 @@
-"# book_store" 
+<img height="200" src="book_store_image.png" width="500" style="display: block; margin: 0 auto; margin-top: 1pc;">
+<h1 style="text-align: center; font-size: 36px; margin-top: -0.5pc;">Book store</h1>
+<h1 style="font-size: 24.5px; margin-top: -2pc; margin-left: 0pc;">Introduction</h1>
+<p style="font-size: 16px; margin-top: -0.8pc;">Book Store is an online platform created for user convenience in searching, browsing, and purchasing books through the Internet. It solves several issues:</p>
+<ul style="font-size: 16px; padding-left: 20px; margin-top: -0.8pc; margin-left: 0pc;">
+    <li>Convenient book search: Users can search for books by title and author, allowing them to quickly find the desired books.</li>
+    <li>Book purchasing: Users have the option to purchase books directly through this platform, conveniently selecting desired books and putting them into a shopping cart.</li>
+</ul>
+<h1 style="font-size: 24.5px; margin-top: -1pc; margin-left: 0pc;">Technology stack</h1>
+<p style="font-size: 16px; margin-top: -0.8pc;">While working on the project, I used the following technologies: <span style="color: black; font-weight: bold;">Spring Boot, Spring Security, Spring Data JPA, Spring MVC, Swagger, Java, Hibernate, MySQL, Spring, Maven, JUnit, Mockito, Windows, Servlets, Tomcat, JSP, XML, JSON, Git, IDE.</span></p>

```

Comment:
Please describe the stack in more detail, what is the purpose of the technologies that you used?